SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -226,21 +229,74 @@ void USpatialGDKEditorSettings::SetNumberOfSimulatedPlayers(uint32 Number)
 	SaveConfig();
 }
 
-bool USpatialGDKEditorSettings::IsManualWorkerConnectionSet(const FString& LaunchConfigPath)
+bool USpatialGDKEditorSettings::IsManualWorkerConnectionSet(const FString& LaunchConfigPath, TArray<FString>& OutWorkersManuallyLaunched)
 {
-	FString FileContents;
-	FFileHelper::LoadFileToString(FileContents, *LaunchConfigPath);
+	TSharedPtr<FJsonValue> Document;
+	{
+		FArchive* ConfigFile = IFileManager::Get().CreateFileReader(*LaunchConfigPath);
+
+		if (!ConfigFile)
+		{
+			UE_LOG(LogSpatialEditorSettings, Error, TEXT("Could not open configuration file %s"), *LaunchConfigPath);
+			return false;
+		}
+
+		auto JsonReader = TJsonReader<char>::Create(ConfigFile);
+
+		FJsonSerializer::Deserialize(*JsonReader, Document);
+
+		delete ConfigFile;
+		ConfigFile = nullptr;
+	}
```

Comment:
```suggestion
	}
	
```
I think we usually have a newline after `}`